README.md

```diff
@@ -1,74 +1,190 @@
-# Hello Microverse
 
-**Hello Microverse** project is a project based on implementing new features on future projects.
 
-# Built With
-  Client:
-  * HTML.
-  * CSS.
-  
-  Server:
-  * none
+# 📖 Hello Microverse 
 
-Key Features
 
- **Text styling**
+> Describe your project in 1 or 2 sentences.
+
+**Hello Microverse** is a project created to add new features.
+
+## 🛠 Built With HTML and CSS
+
+### Tech Stack
+
+
+
+ **Client**
+
+   [HTML](https://www.w3.org/)
+
+
+
+  **Styling**
+
+  [CSS](https://web.dev/learn/css/)
+
+
+**Database**
+
+   [MySQL](https://www.mysql.com/)
+
```

Comment:
- [ ] Good job trying to update the stacks, I see you forgot to remove the database, please make sure to remove it since there is no database for this project :recycle: